jujugui/static/gui/src/app/jujulib/bundleservice.js

```diff
@@ -0,0 +1,60 @@
+/* Copyright (C) 2016 Canonical Ltd. */
+'use strict';
+
+var module = module;
+
+(function (exports) {
+
+  var jujulib = exports.jujulib;
+
+  /**
+    bundleservice API Client
+  */
+
+  /**
+    Initializer.
+
+    @function bundleservice
+    @param url {String} The URL of the bundleservice instance, including
+      scheme, port and the trailing slash, and excluding the API version
+      if applicable.
+    @returns {Object} A client object for making bundleservice API calls.
+  */
+  function bundleservice(url, webhandler) {
+    // Store the API URL handling trailing slash.
+    this.url = url.replace(/\/?$/, '');
+    this.webhandler = webhandler;
+  };
+
+  bundleservice.prototype = {
+    /*
+      Gets the bundle changes from the bundle YAML.
+
+      @method getBundleChangesFromYAML
+      @param {String} bundleYAML The bundle contents in YAML format.
+      @param {Function} callback The callback function to call with the
```

Comment:
Please described what's included in the response. Also, the callback here actually receives an error and the response.